### ISSUE TITLE
Got u'Printed-' between two lines. Expected nothing or a preserved line break

### ISSUE THREAD

**Issue by kingshukb** [Kozea/WeasyPrint#417]
This issue is similar, to #389, however i am 
getting this error while trying to create pdf from html:

from weasyprint import HTML, CSS
 HTML(string=output).write_pdf(pdf_file_name, stylesheets=[CSS(string=css_string)])

I have attached the stack-trace in detail:
[Weasy_error.txt](https://github.com/Kozea/WeasyPrint/files/742225/Weasy_error.txt)

I am using : 
WeasyPrint==0.34

Thank you for providing WeasyPrint!

**Comment by liZe** [comment]
> Thank you for providing WeasyPrint!

Thank you for reporting :wink:!

Could you please provide your HTML and CSS samples?

**Comment by liZe** [comment]
I think that it's been fixed by #410, could you please try the current master branch?

**Comment by liZe** [comment]
No news is good news, I'm pretty sure that it's fixed now.

**Comment by jsatt** [comment]
I don't think this is actually fixed. I'm getting a similar error on our production servers.

weasyprint 0.36
```python
weasyprint.HTML(string=<<string in attached file>>).write_pdf('/tmp/raw_test.pdf')
```
[html and css used](https://github.com/Kozea/WeasyPrint/files/1002553/weasysample.txt)
[traceback](https://github.com/Kozea/WeasyPrint/files/1002555/weasyerror.txt)

**Comment by liZe** [comment]
It's now fixed (duplicate of #449).
